doc/breaking_change_rules/1007.md

```diff
@@ -0,0 +1,49 @@
+### 1007 - Cmd Parameter Removed
+
+**Description**: Checks whether an existing parameter of a cmd is removed from the previous cli version. 
+
+**Analysis**: This is considered a breaking change because it may influence users' current workflow. This change requires to be released in breaking-change window.
```

Comment:
```suggestion
**Analysis**: This is considered a breaking change because it may influence users' current workflow. This change requires to be released during breaking-change window.
```